lib/evaluation/abstractEvaluationHandler.ts

```diff
@@ -1,11 +1,18 @@
 import { PresentationDefinition } from '@sphereon/pe-models';
 
+import { EvaluationClient } from './evaluationClient';
 import { EvaluationHandler } from './evaluationHandler';
 import { HandlerCheckResult } from './handlerCheckResult';
 
 export abstract class AbstractEvaluationHandler implements EvaluationHandler {
   private nextHandler: EvaluationHandler;
 
+  private _client: EvaluationClient;
+
+  constructor(client: EvaluationClient) {
```

Comment:
The backing field for the property is automatically created. You just need to write: constructor(_client: EvaluationClient) {}

Reply:
changed